source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,261 @@
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+      - command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+    outcome:
+        collection:
+            data:
+                - _id: 1
+
+  - description: commitTransaction retries once after a network exception
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+        failCommands: ["commitTransaction"]
+        closeConnection: true
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+            _id: 1
+        result:
+        insertedId: 1
+      - name: commitTransaction
+        object: session0
+    expectations:
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+            - _id: 1
+            ordered: true
+            readConcern:
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction: true
+            autocommit: false
+            writeConcern:
+          command_name: insert
+          database_name: *database_name
+      - command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+      - command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber:
+            $numberLong: "1"
+            startTransaction:
+            autocommit: false
+            writeConcern:
+          command_name: commitTransaction
+          database_name: admin
+    outcome:
+      collection:
+        data:
+          - _id: 1
+
+  - description: commitTransaction does not retry error without RetryableWriteError label
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+        failCommands: ["commitTransaction"]
+        errorLabels: []
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+          _id: 1
+        result:
+        insertedId: 1
```

Comment:
Indentation is off here. Please take a pass through everything to see if this mistake was made elsewhere.